frontend/src/components/Item/SeasonTabs.vue

```diff
@@ -5,34 +5,42 @@
         {{ season.Name }}
       </v-tab>
     </v-tabs>
-    <v-tabs v-model="currentTab" class="bg-transparent">
-      <v-tab v-for="season in seasons" :key="season.Id">
+    <v-window v-model="currentTab" class="bg-transparent">
+      <v-window-item v-for="season in seasons" :key="season.Id">
         <v-list
           v-if="seasonEpisodes && season.Id"
-          lines="two"
-          color="transparent">
+          lines="false"
+          bg-color="transparent">
           <v-list-item
             v-for="episode in seasonEpisodes[season.Id]"
             :key="episode.Id"
-            :to="getItemDetailsLink(episode)"
-            class="flex-column flex-md-row">
-            <v-avatar width="20em" height="12em">
-              <blurhash-image
-                v-if="episode.ImageTags && episode.ImageTags.Primary"
-                :item="episode"
-                :alt="episode.Name" />
-              <watched-indicator v-if="episode.UserData.Played" />
-            </v-avatar>
-            <v-list-item-title class="text-wrap">
-              {{ episode.IndexNumber }}. {{ episode.Name }}
-            </v-list-item-title>
-            <v-list-item-subtitle class="text-wrap">
-              {{ episode.Overview }}
-            </v-list-item-subtitle>
+            :to="getItemDetailsLink(episode)">
+            <v-row class="mb-8">
+              <v-col
+                :class="{ 'py-1': $vuetify.display.smAndDown }"
+                cols="12"
+                md="4">
+                <card
+                  :class="{ 'mx-8 mt-8': $vuetify.display.smAndDown }"
+                  :item="episode" />
+                <watched-indicator v-if="episode.UserData?.Played" />
```

Comment:
Not needed anymore as the indicator is added by the card, and this just adds the indicator twice on the right of the item
```suggestion
```